python/cudf/dataframe/categorical.py

```diff
@@ -112,11 +112,11 @@ def cat(self):
                                    ordered=self._ordered)
 
     def binary_operator(self, binop, rhs):
-        msg = 'Categorical cannot perform the operation: {}'.format(binop)
+        msg = 'Series cannot perform the operation: {}'.format(binop)
```

Comment:
Not a big deal, but wouldn't `'Categorical Series cannot perform the operation: {}'` be slightly more helpful?

Reply:
I'd be for changing the message to be more helpful since I don't think this discrepancy would affect dask or anything like that, but I leave it totally to your judgement call